Foundation/Data.swift

```diff
@@ -790,7 +790,7 @@ public final class _DataStorage {
     }
 }
 
-internal class _NSSwiftData : NSData {
+public class _NSSwiftData : NSData {
```

Comment:
`DEPLOYMENT_RUNTIME_SWIFT` is defined for swift-corelibs-foundation and is not for the overlay since it's runtime is objc.